test/unit/services/CryptoService.spec.js

```diff
@@ -16,29 +16,112 @@
  */
 
 import chai from "chai";
-chai.should();
+const should = chai.should();
 
-import { createHmac, hmacLoc } from '../../../src/services/CryptoService';
-import { zucotti } from '../../support/sampleLocations';
-import { zucottiHmac } from '../../support/cryptoStrings';
+import { encrypt, decrypt, sign, verify, getKey } from '../../../src/services/CryptoService';
+import { heyWhereAtCT } from '../../support/cryptoStrings';
+
+import sjcl from "../../../lib/sjcl";
+const { 
+  codec: { utf8String: { toBits }, hex: { fromBits } }, 
+  misc: { hmac } 
+} = sjcl;
 
 describe('CryptoService', () => {
 
-  describe('createHmac', () => {
+  describe('encrypt', () => {
+
+    it('returns encrypted json string', () => {
+      let key = toBits('shared secret');
+      encrypt(key, 'hey, where at?', {"iv":"yhgvpSW8b+oPkHvHGlMJCQ=="}).should.eql(heyWhereAtCT);
+    });
+
+  });
+
+  describe('decrypt', () => {
+    
+    it('decrypts provided string', () => {
+      let key = toBits('shared secret');
+      decrypt(key,heyWhereAtCT).should.eql('hey, where at?');
+    });
+
```

Comment:
strike my previous comment: this is likely a better place to add a round trip test. something like...

```javascript
decrypt(key, encrypt(key, 'foo', {'iv': 'bar'})).should.eql('foo');
```